packages/cli/src/__tests__/commands/__snapshots__/Version.test.ts.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`version basic version (Node-API) 1`] = `
 prisma                  : 0.0.0
-@prisma/client          : 0.0.0
+@prisma/client          : Not found
```

Comment:
Change needed because of the migration to `pnpm@7`
https://prisma-company.slack.com/archives/CUXLS0Z6K/p1655916241631919?thread_ts=1655914673.859489&cid=CUXLS0Z6K